09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO <<<<<<<<<<<<<<<<<< Logger setup anew <<<<<<<<<<<<<<<<<<< (utils.py:39) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Analyzing repos/repos_400_c_round_two/saminiir___level-ip (install_repos.py:285) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Checking for build system in repos/repos_400_c_round_two/saminiir___level-ip/src (install_repos.py:267) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Running make (install_repos.py:51) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Running command: make clean (install_repos.py:20) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/saminiir___level-ip_build.log (install_repos.py:21) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG rm build/*.o lvl-ip (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG rm: cannot remove 'build/*.o': No such file or directory (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG rm: cannot remove 'lvl-ip': No such file or directory (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG make: *** [Makefile:36: clean] Error 1 (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Running command: make distclean (install_repos.py:20) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/saminiir___level-ip_build.log (install_repos.py:21) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG make: *** No rule to make target 'distclean'.  Stop. (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Running command: rm -rf autom4te.cache (install_repos.py:20) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/saminiir___level-ip_build.log (install_repos.py:21) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Running command: rm -f config.status config.cache config.log (install_repos.py:20) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/saminiir___level-ip_build.log (install_repos.py:21) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Running autogen (install_repos.py:57) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Running command: ./autogen (install_repos.py:20) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/saminiir___level-ip_build.log (install_repos.py:21) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG /bin/sh: 1: ./autogen: not found (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Running ./configure (install_repos.py:61) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Running command: ./configure (install_repos.py:20) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/saminiir___level-ip_build.log (install_repos.py:21) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG /bin/sh: 1: ./configure: not found (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Running make (install_repos.py:69) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Running command: make (install_repos.py:20) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip INFO Logging output to: logs/repos/repos_400_c_round_two_install_logs_with_make_cmake_gradlew_meson_scons_bazel_2024-09-12_17-35-05/saminiir___level-ip_build.log (install_repos.py:21) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG cc  -I include -Wall -Werror -pthread -c src/arp.c -o build/arp.o (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG In file included from include/arp.h:3, (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG from src/arp.c:1: (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG include/syshead.h:27:10: fatal error: sys/capability.h: No such file or directory (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG 27 | #include <sys/capability.h> (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG |          ^~~~~~~~~~~~~~~~~~ (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG compilation terminated. (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip DEBUG make: *** [Makefile:15: build/arp.o] Error 1 (install_repos.py:38) (saminiir___level-ip)
09/12/2024 05:43:26 PM logger_saminiir___level-ip ERROR Error: Build failed for saminiir___level-ip
Build failed for reason: make failed (install_repos.py:295) (saminiir___level-ip)
